### PR TITLE
remove trailing slash on artifactory server url

### PR DESCRIPTION
**Pull request by Jetersen**
I have done this mistake a few times 😓

### CODE REVIEW

**Comment by MindaugasLaganeckas** [comment]
This is a super great PR! We would very much like to have it merged! :heart:

File: artifactory_cleanup/artifactorycleanup.py [hunk 1]
```diff
@@ -69,6 +69,9 @@ def _destroy_or_verbose(self):
             print("Verbose MODE")
 
     def main(self):
+        if self._artifactory_server.endswith("/"):
```
Comment (allburov) [inline artifactory_cleanup/artifactorycleanup.py]: We can `rstrip`, without `if statement. Rstrip won't do anything if string ends no with  "/".

**Comment by allburov** [comment]
@Jetersen thanks for the contributing! Let's simplify solution and we will pull it.

File: artifactory_cleanup/artifactorycleanup.py [hunk 2]
```diff
@@ -69,6 +69,9 @@ def _destroy_or_verbose(self):
...
+            # remove trailing slash
+            self._artifactory_server = self._artifactory_server.rstrip("/")
```
Comment (Jetersen) [inline artifactory_cleanup/artifactorycleanup.py]: You can always use multiline suggestions nowadays 😄 
```suggestion
        # remove trailing slash
        self._artifactory_server = self._artifactory_server.rstrip("/")
```

**Comment by Tim55667757** [comment]
@Jetersen tnx for PR)

**Pull request merged**